File: spring-web-java/src/main/java/spring/web/java/domain/Notification.java
package spring.web.java.domain;

import jakarta.persistence.*;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Notification extends Base {

    @Id
    @GeneratedValue
    @Column(name = "notification_id")
    private Long id;
    private String type;
    private String title;
    private String message;
    private String url;
}

File: spring-web-kotlin/src/main/kotlin/spring/web/kotlin/domain/Notification.kt
package spring.web.kotlin.domain

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id

@Entity
class Notification private constructor(

    @Id
    @GeneratedValue
    @Column(name = "notification_id")
    val id: Long?,
    val type: String,
    val title: String,
    val message: String,
    val url: String
) : Base()